src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -410,10 +410,10 @@ ss::future<> service::do_start() {
           std::current_exception());
         throw;
     }
-    co_await container().invoke_on_all(_ctx.smp_sg, [](service& s) {
-        s._is_started = true;
-        return s.fetch_internal_topic();
-    });
+    co_await container().invoke_on(
+      0, _ctx.smp_sg, [](service& s) { return s.fetch_internal_topic(); });
+    co_await container().invoke_on_all(
+      _ctx.smp_sg, [](service& s) { s._is_started = true; });
```

Comment:
This is all invoked from a `one_shot`, so it should be safe.

I'd be tempted to do the cross-shard invoke once, though:
```suggestion
    co_await container().invoke_on_all(_ctx.smp_sg, [](service& s) {
        s._is_started = true;
        return ss::this_shard_id() == 0 ? s.fetch_internal_topic() : ss::now();
    });
```